deduplicator/src/bin/main.rs

```diff
@@ -122,8 +132,8 @@ fn main() -> rusqlite::Result<()> {
         load_from_sqlite(
             &mut deduplication,
             path,
-            move |addr| source.filter(&addr),
-            move |addr| source.ranking(&addr),
+            move |addr| source.filter(addr),
+            move |addr| source.ranking(addr),
```

Comment:
Theses lines and the two below are here because clippy became better at detecting irrelevant borrowing.